app/batching/get-agreement-reference.js

```diff
@@ -0,0 +1,15 @@
+const getAgreementReference = (source, agreementNumber) => {
+  // With the exception of SitiELM and SITICS, for sources beginning with Siti, DAX will populate the remittance advice with an invalid reference
+  // This can be avoided by not passing the agreement number on ledger lines
```

Comment:
I assume these comments are being kept?

Reply:
Yeah I think it's best to keep them.  Without them there's no logical reason for why this function does what it does.  I think the explanation is helpful for future devs.

Reply:
Yep, I would agree